### PR TITLE
Fv 2022 07 08

### PR DESCRIPTION
**Pull request by fvance**
Create file for Label 83 and provide examples.
Add additional example to Label 10 and make some edits.

### CODE REVIEW

File: research/10.md [hunk 1]
```diff
@@ -45,4 +55,11 @@ Appears to be a message to Operations.
 
 ## Acronyms / Codes
 
+- KHOU - Houston Hobby Airport
```
Comment (kevinelliott) [inline research/10.md:58]: Not necessary to spell these out, here we just need to identify unique codes. I'd say we can just say identify that there are airport codes, etc.

For example maybe:
```
LDR - whatever this means
KXXX - origin and destination airport
```

File: research/83.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+### Example from registration N885GL, ICAO AC2D52 (not an air carrier)
+Recorded Sunday, June 19, 2022 11:35:24 UTC
+
+*Plane was reporting flight as "XA0001".*
+
+```
+001PR19113523N2947.7W09536.51273750177
+```
+
+- characters 1 through 5 - unknown
```
Comment (kevinelliott) [inline research/83.md:20]: Possible to use the table format I use such as in https://github.com/airframesio/acars-message-documentation/blob/main/research/20/POS.md ? I know it's more work/typing.

File: research/10.md (hunk 1 above)
Comment (fvance) [reply research/10.md:58]: I can update that.

File: research/83.md (hunk 2 above)
Comment (fvance) [reply research/83.md:20]: I will look that over, and make that change.

Comment (fvance) [reply research/83.md]: I note that the End column number is actually the column *after* the end of the field in 20/POS.md.

Just want to verify that is the standard to be used.

**Comment by kevinelliott** [comment]
Awesome @fvance the improvements are much appreciated!

**Pull request merged**